hack/tools/go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/fluxcd/helm-operator/hack/tools
 go 1.13
 
 require (
-	github.com/spf13/pflag v1.0.5 // indirect
 	golang.org/x/tools v0.0.0-20190927052746-69890759d905 // indirect
-	k8s.io/code-generator v0.16.5-beta.1 // kubernetes-1.16.2
+	k8s.io/code-generator v0.17.0 // kubernetes-1.16.2
```

Comment:
I think the generated code and the controller should use the same Kubernetes version, otherwise we risk to have an incompatible clientset.